src/components/Button/types.ts

```diff
@@ -0,0 +1,4 @@
+export const PRIMARY_TYPE = 'primary';
+export const SECONDARY_TYPE = 'secondary';
+
+export type ButtonType = typeof PRIMARY_TYPE | typeof SECONDARY_TYPE;
```

Comment:
Lets rename it to `ButtonVariantType`
```suggestion
export type ButtonVariant = typeof PRIMARY_TYPE | typeof SECONDARY_TYPE;
```

Reply:
accepted and changed